invenio_records_marc21/services/services.py

```diff
@@ -85,6 +90,47 @@ def update_draft(
         data = self._create_data(identity, data, metadata, access)
         return super().update_draft(id_, identity, data, revision_id)
 
+    def _lift_embargo_from(self, record):
+        """Lifts embargo from record or draft."""
+        if not record.access.embargo.lift():
+            raise EmbargoNotLiftedError(record["id"])
+        record.access.protection.metadata = "public"
+        record.access.protection.files = "public"
+
+    def _draft_access_field_was_modified(self, draft, record):
+        """Returns True if draft's access field was modified."""
+        return draft.get('access') == record.get('access')
+
+    def lift_embargo(self, _id, identity):
+        """Lifts embargo from the record and updates draft."""
+        # Get the record
+        record = self.record_cls.pid.resolve(_id)
+
+        # Check permissions
```

Comment:
the comment is somehow misleadingly, a check indicates that the function returns true or false, but than the function call would be useless. the definition of the function indicates that if the sub function call returns false it throws a error. Better to remove the comment